inference-engine/tests/functional/plugin/shared/src/single_layer_tests/lstm_cell.cpp

```diff
@@ -32,11 +31,11 @@ std::string LSTMCellTest::getTestCaseName(const testing::TestParamInfo<LSTMCellP
     float clip;
     InferenceEngine::Precision netPrecision;
     std::string targetDevice;
-    std::tie(should_decompose, batch, hidden_size, input_size, activations, clip, netPrecision,
-            targetDevice) = obj.param;
+    std::tie(should_decompose, batch, hidden_size, input_size, activations, activations_alpha, activations_beta, clip,
```

Comment:
I think since alpha & beta are unused, it would be OK to not add them here.

No harm in adding them though, so it's also OK to leave this code as-is.

Reply:
I have reverted tests code to without activations alpha beta version to make it simple. 
We can add them later if needed.